### DISCUSSION TITLE
Make keycount easier to search and filter in beatmap listing.

### DISCUSSION THREAD

**Discussion by swetdep** [ppy/osu#19745] [Ideas] [answered]
Keycount is one of the hardest things to search, when you're looking for ranked maps, and the only 3 options you have are to search for different posts in reddit, forums or similar, to download beatmap packs, or to simply search the keycount in the beatmap listing (in which, only beatmaps with difficulties named as the keycount, e.g. 6k insane will appear).

Here are two ways to approach easier keycount searching:
-Add keycount as a filter in the beatmap listing after setting "mania" as a mode, or when "mania" and "any" are selected. 
This make keycount searching way easier, and might make more people search for rare keycounts (such as 5, 6, 10, or even 3) more oftenly.
-Add a filtering term for searching specific keys (like searching for bpm in stable). This won't make it easier for more people to get into different keycounts, but it wouldn't change the interface. Of course might make searching for unique difficulty names, but these are very rare.

**Comment by smoogipoo** [comment, marked as answer]
Does searching `keys=6` not work? https://osu.ppy.sh/beatmapsets?q=keys%3D6

**Reply by swetdep** [reply]
_this is very awkward... it works_, 
but even if it does, I still would really like to it to be the first option I gave, as it is still way easier to find about...
_(and i'm very sorry not actually knowing you could do that, although there was no way for me to know about it)_

**Reply by smoogipoo** [reply]
I've made an issue for you here: https://github.com/ppy/osu-web/issues/9203
